### DISCUSSION TITLE
按需引入，Vetur与ts-plugin提示：找不到名称“showToast”

### DISCUSSION THREAD

**Discussion by Comfident** [youzan/vant#13247] [Q&A]
您好，我使用的ide为vscode。

官方文档中：“在 <script setup> 中可以直接使用 Vant 组件”效果很不错，但是我有个小困惑

在不import组件的情况下，Vetur与ts-plugin检查对于 showToast，showConfirmDialog等不识别

![image](https://github.com/user-attachments/assets/14924882-d05c-4a5d-b76b-9132c1dbfca3)

请问我该如何取消这种报错，期待大家讨论，蟹蟹~

**Comment by coderBianer** [comment]
https://github.com/youzan/vant/discussions/12775
